front/src/applications/operationalStudies/components/SimulationResults/SimulationResultsMap.tsx

```diff
@@ -149,7 +151,7 @@ const Map: FC<MapProps> = ({ setExtViewport }) => {
         const trainTime = trainRegime.head_positions[0][0].time;
         const train2ndTime = last(last(trainRegime.head_positions))?.time as number;
         if (
-          actualTime >= trainTime &&
+          actualTime >= (trainTime as number) &&
```

Comment:
Are you sure you need this cast ? I removed it and no warning appeared
```suggestion
          actualTime >= trainTime &&
```

Reply:
Done